cmake/Reprojucer.cmake

```diff
@@ -1622,33 +1635,51 @@ function(jucer_project_end)
         unset(auv3_target)
       endif()
 
-      if(TARGET ${target}_AUv3_AppExtension)
-        set(standalone_target ${target}_AUv3_Standalone)
-        add_executable(${standalone_target} MACOSX_BUNDLE
+      if(JUCER_BUILD_AUDIOUNIT_V3
+          AND DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)
+        set(juce4_standalone ON)
+      endif()
+
+      if(juce4_standalone OR (JUCER_BUILD_STANDALONE_PLUGIN
+          AND NOT (DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)))
```

Comment:
Could you make using `JUCER_BUILD_STANDALONE_PLUGIN` an error when using Juce 4 (it's already a warning now)? You wouldn't need this `AND NOT (DEFINED ...)` part of the expression here.

Reply:
I could, but I don't really want to. I want to allow users to go back and forth between JUCE 4 and JUCE 5 if they want, without FRUT getting in their way.